src/element/textWysiwyg.test.tsx

```diff
@@ -837,25 +871,14 @@ describe("textWysiwyg", () => {
         { id: text.id, type: "text" },
       ]);
 
-      // Attempt to bind another text
-      UI.clickTool("text");
-      mouse.clickAt(
-        rectangle.x + rectangle.width / 2,
-        rectangle.y + rectangle.height / 2,
-      );
-      mouse.down();
-      expect(h.elements.length).toBe(3);
-      text = h.elements[2] as ExcalidrawTextElementWithContainer;
-      editor = document.querySelector(
-        ".excalidraw-textEditorContainer > textarea",
-      ) as HTMLTextAreaElement;
-      await new Promise((r) => setTimeout(r, 0));
-      fireEvent.change(editor, { target: { value: "Whats up?" } });
-      editor.blur();
```

Comment:
why remove this test?

Reply:
This test is somewhat incorrect since if a container has bound text then clicking on the center should always edit that text instead of creating a separate text element.
I had spent some time debugging this test, I will add a separate for text tool later. I am not sure why this test was working earlier, probably coz the coords were slightly different when using text tool and since width was too small (1px) so the new text was treated outside container and now since container is being computed earlier so this test started failing.  In any case this test is actually not correct so I modified it.

Reply:
checking it out (with modified assertions), it seems not only a new text is being created, but it's also being bound to the container. It can mean the test conditions are weird / test is not working correctly, or it can mean the implem code incorrectly allows for some edge case that it shouldn't (possibly both, since I can't repro in browser). So I wouldn't totally brush this off, and rather investigate this (put logs into the `startTextEditing` handlers to see why the test is creating a new text, etc.).

Reply:
it's actually due to the initial bound text being marked as deleted (due to `0` dimensions I guess). But even fixing that still breaks it, possibly the hitbox test. So screw it. JSDOM is simply too annoying.